### ISSUE TITLE
normalize fmri data before input to GCN

### ISSUE THREAD

**Issue by zhangyu2ustc** [SIMEXP/fmri_predict#22]
for fmri data, do we need to normalize it to [-1,1] or [0,1] and how to do that?

previously, all data processing including scale/normalization was done in numpy/sklearn
but I am not sure whether it actually helps in pytorch nn.Modules

my experiments shows that, after disable the fmri_data_transform, the performance of MLP(4 layers) jumps from 0.79 to 0.87
